src/developer-documentation/keys/keys.md

```diff
@@ -0,0 +1,43 @@
+# Creating Keys
+
+In GGX chain, key generation is a crucial process that allows users to create cryptographic key pairs. These key pairs consist of a public key and a corresponding private key. The public key is used to derive the account address, while the private key is essential for signing transactions and proving ownership of the account. GGX supports multiple methods for key generation, including using the [GGX Explorer UI](ggx-explorer-create-keys.md), [Polkadot.js extension](developer-documentation/keys/polkadot-js-create-keys.md), [Subkey tool](developer-documentation/keys/subkey-create-keys.md), [JS API](polkadot-js-create-keys.md), and direct generation from the [GGX Node](developer-documentation/keys/node-create-key.md). For better security and advanced users, it is recommended to use node-based methods or the Subkey tool. Multisig and Proxied accounts can be easily created using the [Explorer UI](ggx-explorer-create-keys.md). EVM-based keys can be created by using Metamask and other EVM-compatible wallets.
+
+## Supported Key Standards
+
+GGX chain supports the following key standards for its accounts:
+
+##### ed25519
+This format **is required** for running a validator node. However, conversion between sr25519 and ed25519 is not straightforward since they are different elliptic curve systems. They have different mathematical foundations and converting a private key from one to the other can compromise security. Therefore, it's recommended to use ed25519 keys for validator nodes and sr25519 or EVM-based keys for end users.
```

Comment:
```suggestion
This format **is required** for running a validator node. However, conversion between sr25519 and ed25519 is not straightforward since they are different elliptic curve systems. They have different mathematical foundations and converting a private key from one to the other can compromise security. Therefore, it is recommended to use ed25519 keys for validator nodes and sr25519 or EVM-based keys for end users.
```